wiki/Quality_Assurance_Team_Blog/fr.md

```diff
@@ -0,0 +1,72 @@
+# Blog de l'équipe d'assurance qualité
+
+Le **[Blog de l'équipe d'assurance qualité](http://osuqat.tumblr.com)**, communément appelé le * Blog QAT *, était un blog semi-régulier écrit par des membres du [Equipe Assurance Qualité](/wiki/Glossary#quality-assurance-team). À partir d'août 2016, ils ont discuté non seulement des activités de la QAT, mais également de nombreux autres sujets couvrant le gameplay d'osu!, mapping, modding et le jeu/la communauté dans son ensemble.
```

Comment:
```suggestion
Le **[Blog de l'équipe d'assurance qualité](http://osuqat.tumblr.com)**, communément appelé le *Blog QAT*, était un blog semi-régulier écrit par des membres du [Equipe Assurance Qualité](/wiki/Glossary#quality-assurance-team). À partir d'août 2016, ils ont discuté non seulement des activités de la QAT, mais également de nombreux autres sujets couvrant le gameplay d'osu!, mapping, modding et le jeu/la communauté dans son ensemble.
```
the "commonly referred to" note may not be needed if this doesn't apply to the French osu community. not sure if you considered that when translating or just 1:1 copied english